content/docs/user-guide/project-structure/internal-files.md

```diff
@@ -100,6 +70,34 @@ Not to be confused with `.dvc` files.
 [queued experiments]:
   /doc/user-guide/experiment-management/running-experiments#the-experiments-queue
 
+## Site cache dir
```

Comment:
Feels like this is a little out of place.

Reply:
Not sure it's needed, but if we keep it, I'd vote to move to the bottom of this page, even if it logically "belongs" here.

Reply:
Moved, thanks!